cli/args/flags.rs

```diff
@@ -5620,7 +5620,7 @@ mod tests {
 
   #[test]
   fn allow_read_allowlist() {
-    use test_util::TempDir;
+    use test_server::TempDir;
```

Comment:
Since a lot of these will probably need to be changed back once we split test_server up, let's do this:

 - In all the crates that use test_util, import it like so:

`test_util = { package = "test_server", workspace = true }`

 - Revert the changes from `use test_util` to `use test_server`

We can restore these in a later patch when we split the util crate from the server crate, and the patch won't be quite as large.